src/main/resources/META-INF/native-image/jansi-native/resource-config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "resources": [
+    {"pattern": "META-INF/native/windows64/jansi.dll"}
```

Comment:
this is not the only resource: please add other that may be useful (not sure windows32 for example is useful)